app/integrations/discord/interactions/resolvers/resolver.rb

```diff
@@ -0,0 +1,51 @@
+module Discord
+  module Interactions
+    module Resolvers
+      class Resolver
+        class << self
+          def create_resolver(request:)
+            Resolvers::Ping.new(request:) if request.ping_type?
+          end
+        end
+
+        PUBLIC_KEY = ENV['PUBLIC_KEY']
+        RESPONSE_STATUS = {
+          OK: 200,
+          UNAUTHORIZED: 401
+        }.freeze
```

Comment:
In ruby only the constants are written in uppercase, in these hashes that you use to build, they keys are symbols, not constants so it makes things confusing when you use uppercase for the keys

```suggestion
        RESPONSE_STATUS = {
          ok: 200,
          unauthorized: 401
        }.freeze
```

Reply:
Done!